backend/api/Models/Models.cs

```diff
@@ -7,7 +7,7 @@ namespace api.Models
     public class Robot
     {
         [Key]
-        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        //[DatabaseGenerated(DatabaseGeneratedOption.Identity)]
```

Comment:
Ideally I would not comment out this line and make the test work with autogenerated IDs. Also remove the IDs from the InitDb.cs file, as we always want the DB to create the IDs

Reply:
Fixed